corehq/messaging/scheduling/scheduling_partitioned/tests/test_dbaccessors_non_partitioned.py

```diff
@@ -0,0 +1,245 @@
+import uuid
+from corehq.form_processor.tests.utils import only_run_with_non_partitioned_database
+from corehq.messaging.scheduling.scheduling_partitioned.dbaccessors import (
+    get_alert_schedule_instance,
+    get_timed_schedule_instance,
+    save_alert_schedule_instance,
+    save_timed_schedule_instance,
+    delete_alert_schedule_instance,
+    delete_timed_schedule_instance,
+    get_active_alert_schedule_instance_ids,
+    get_active_timed_schedule_instance_ids,
+    get_alert_schedule_instances_for_schedule,
+    get_timed_schedule_instances_for_schedule,
+)
+from corehq.messaging.scheduling.models import (
+    AlertSchedule,
+    TimedSchedule,
+)
+from corehq.messaging.scheduling.scheduling_partitioned.models import (
+    AlertScheduleInstance,
+    TimedScheduleInstance,
+)
+from corehq.util.exceptions import AccessRestricted
+from datetime import datetime, date
+from django.conf import settings
+from django.test import TestCase
+
+
+@only_run_with_non_partitioned_database
```

Comment:
this isn't needed when running with travis since there are different jobs for partitioned vs non partitioned, but it makes a difference when running locally. without it, it would try to run these tests even if i'm running with partitioned settings